packages/govuk-frontend/src/govuk/components/accordion/accordion.mjs

```diff
@@ -256,23 +255,25 @@ export class Accordion extends GOVUKFrontendComponent {
     // Create container for heading text so it can be styled
     const $headingText = document.createElement('span')
     $headingText.classList.add(this.sectionHeadingTextClass)
-    // Copy the span ID to the heading text to allow it to be referenced by `aria-labelledby` on the
-    // hidden content area without "Show this section"
+    // Copy the span ID to the heading text to allow it to be referenced by
+    // `aria-labelledby` on the hidden content area without "Show this section"
     $headingText.id = $span.id
 
-    // Create an inner heading text container to limit the width of the focus state
+    // Create an inner heading text container to limit the width of the focus
+    // state
     const $headingTextFocus = document.createElement('span')
     $headingTextFocus.classList.add(this.sectionHeadingTextFocusClass)
     $headingText.appendChild($headingTextFocus)
-    // span could contain HTML elements (see https://www.w3.org/TR/2011/WD-html5-20110525/content-models.html#phrasing-content)
+    // span could contain HTML elements
+    // (see https://www.w3.org/TR/2011/WD-html5-20110525/content-models.html#phrasing-content)
     $headingTextFocus.innerHTML = $span.innerHTML
 
     // Create container for show / hide icons and text.
     const $showHideToggle = document.createElement('span')
     $showHideToggle.classList.add(this.sectionShowHideToggleClass)
-    // Tell Google not to index the 'show' text as part of the heading
-    // For the snippet to work with JavaScript, it must be added before adding the page element to the
-    // page's DOM. See https://developers.google.com/search/docs/advanced/robots/robots_meta_tag#data-nosnippet-attr
+    // Tell search engines not to index the 'show' text as part of the heading.
```

Comment:
*issue* I think that data-nosnippet is actually a Google thing. All I can find regarding Bing support is [a tweet from last year saying they don't support it, through a plethora of SEO articles](https://duckduckgo.com/?q=bing+data-nosnippet&t=newext&atb=v341-1). Has there been a confirmation of support since?